src/Microsoft.AspNetCore.Blazor.Build/targets/Blazor.MonoRuntime.targets

```diff
@@ -168,6 +168,14 @@
       </BlazorItemOutput>
     </ItemGroup>
 
+    <ItemGroup Label="Static content supplied by NuGet packages">
+      <_BlazorPackageContentOutput Include="@(BlazorPackageContentFile)" Condition="%(SourcePackage) != ''">
+        <TargetOutputPath>$(ProjectDir)$(OutputPath)$(BaseBlazorPackageContentOutputPath)%(SourcePackage)\%(RecursiveDir)\%(Filename)%(Extension)</TargetOutputPath>
+        <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
I'm not sure this does anything on custom items. I think this is a feature of Content. This is good for now, but in the future I think we should move users to use Content for this (as it has all the features). I would need to know more about what we are trying to do here to better judge it, but I'm going to make 2 assumptions.
1) Content from 3rd party packages is immutable.
2) Just needs to be copied to the output without any prior processing.

Reply:
`BlazorPackageContentFile` items are those specifically nominated by the package for inclusion in the `dist` dir.

> Content from 3rd party packages is immutable.

Yes

> Just needs to be copied to the output without any prior processing.

Pretty much, but:

1. We want to output it to `dist`, not just `$(OutDir)` (and don't want package authors to hardcode the notion of `dist` in their packages)
2. We want to strongly guide package authors to give us their `SourcePackage` metadata so that we can drop content into `dist/_content/(SourcePackage)/...` to ensure there are no clashes across packages. The implementation here means you have to give a nonempty `SourcePackage` otherwise we won't include your item in `dist`.